cmd/zoekt-sourcegraph-indexserver/queue.go

```diff
@@ -99,13 +111,21 @@ func (q *Queue) AddOrUpdate(opts IndexOptions) {
 		item.opts = opts
```

Comment:
This options diff is an opportunity to clear backoff state and allow indexing on next enqueue request.